### PR TITLE
resources: Finalize automated generation of the Docker image

### LINKED ISSUE #953: Automatically generate a new Docker image when a PR changes our Dockerfile
And push it to dockerhub.

### PR DESCRIPTION
**Pull request by sboeuf**
By fixing the Dockerfile, we have now finalized the automated generation
of the Docker images for both architectures (amd64 and arm64).

Fixes #953

Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>

### CODE REVIEW

**Comment by sboeuf** [comment]
@rbradford @sameo this PR is ready, but please wait to merge it until we get the green light from https://github.com/cloud-hypervisor/cloud-hypervisor/pull/1742.
The reason is, I've already updated the Docker images from my personal repo, so I know this current PR is the exact same code I used, but before we merge it, I want to see a PR using `latest` passing the CI.

**Comment by sboeuf** [comment]
@rbradford @sameo ok I can confirm the PR #1742 passed properly yesterday, so we can go ahead and merge this PR if you're okay with it.

**Review by rbradford — APPROVED** [review]

**Pull request merged**
